src/AI/CoreLogic.pm

```diff
@@ -1801,7 +1801,14 @@ sub processAutoBuy {
 	}
 
 	if (AI::action eq "buyAuto" && AI::args->{'done'}) {
-		
+		my $args = AI::args;
+		if (exists $args->{recv_buy_packet_owg}){
+			delete $args->{recv_buy_packet_owg};
+			delete $args->{recv_buy_packet};
+			AI::dequeue;
+			AI::queue("sellAuto", {forcedByBuy => 1});
```

Comment:
you should check if sellAuto is activated befofe queue

Reply:
openkore don't queue autobuy, autosell and auto storage together.

Reply:
only if user use macro and force sellAuto in midle of buyAuto